package.json

```diff
@@ -134,6 +135,8 @@
     "globals": "^15.11.0",
     "jest": "^29.7.0",
     "jest-expect-message": "^1.1.3",
+    "jsdom": "^24.1.0",
```

Comment:
Is jsdom still needed? I don't see it imported anywhere.

Reply:
Doesn't look like it, I've removed it now.